src/js/intlTelInput.js.ejs

```diff
@@ -429,7 +429,7 @@ Plugin.prototype = {
     var that = this;
 
     // check for cookie
-    var cookieAutoCountry = ($.cookie) ? $.cookie("itiAutoCountry") : "";
+    var cookieAutoCountry = (!!Cookies.get) ? Cookies.get("itiAutoCountry") : "";
```

Comment:
If Cookies is not defined this will error. Instead you should check if (window.Cookies)